CONTRIBUTING.md

```diff
@@ -0,0 +1,373 @@
+the project on your own GitHub account and make your `master` branch to track this
+repository. A typical Git setup after
+[forking the project](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/fork-a-repo) is:
+
+```sh
+# After forking the repository in GitHub
+git clone https://github.com/<your_username>/qiskit.org.git
+cd qiskit.org
+git remote add upstream https://github.com/qiskit/qiskit.org.git
+git remote update upstream
+git checkout master
+git branch -u upstream/master
+git pull
+```
+
+### Working on an issue
+
+When you are going to start working on an issue, make sure you are in your `master`
+branch and that it is entirely up to date and create a new branch with a
+meaningful name. The typical terminal code for this is:
+
+```sh
+git checkout master
+git pull upstream master
+git checkout -b issue-1234-new-header
+```
+
+Now start adding your changes and remember to commit often:
+
+```sh
+git commit
+```
+
+And include a summary and some notes or clarifications if needed:
+
+```
+Create a new header layout.
+
+Includes a new component factoring out the header of this new page and others.
+```
+
+From time to time, you want to check if your `master` branch is still up to
+date. If not, you will need to
+[rebase](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase) and
+continue working:
+
+```sh
+git checkout master
+git pull
+git checkout issue-1234-new-header
+git rebase -i master
+```
+
+Use the interactive `-i` option of rebase to polish the history of your branch.
+
+### Feature branches
+
+Very often, implementing a user story implies publishing several features in an atomic
+batch. When it happens, we use a feature branch in the main repository for integrating
+all the needed changes before merging to `master`.
+
+When working against a feature branch, you consider the feature branch to be the
+`master` branch and so, when about to start a new task, you do:
+
+```sh
+git checkout feature-branch
+git pull upstream feature-branch
+git checkout -b issue-1234-new-header
+```
+
+And keep an eye on `feature-branch` to always rebase your code on top of the most
+updated version of the `feature-branch`:
+
+```sh
+git checkout feature-branch
+git pull upstream feature-branch
+git checkout issue-1234-new-header
+git rebase -i feature-branch
+```
+
+### Pull requests
+
+Pull requests serve a double purpose:
+1. Share the code with the team. So almost everybody is aware of how the code base is evolving.
+2. Provide an opportunity for improving code quality.
+
+When you think your work is done, push the branch to your repository:
+
+```sh
+git push origin issue-1234-new-header
+# Start a pull request in GitHub
+```
+
+And
+[create a pull request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
+against `master` (or a feature branch).
+When creating the pull request, provide a description and
```

Comment:
If we add here the info related to how we move the cards on the project backlog, here we should say that when we create the PR, we add the card to the "review in progress" column

Reply:
Right, it will go in the Wiki too.